de-lib/src/main/java/org/iplantc/de/apps/client/presenter/hierarchies/OntologyHierarchiesPresenterImpl.java

```diff
@@ -371,4 +454,12 @@ void fireEvent(GwtEvent<?> event) {
             handlerManager.fireEvent(event);
         }
     }
+
+    List<FilteredHierarchyCallback> createFilteredHierarchyList() {
+        return Lists.newArrayList();
+    }
+
+    Promise getCallbackPromise(List<FilteredHierarchyCallback> childCallbacks) {
```

Comment:
Oh sure - this was a last minute change to make the tests work with GQuery so I didn't put much thought into it.